core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -238,6 +245,8 @@ struct SchedulerCountMetrics {
 
     /// Number of transactions scheduled.
     num_scheduled: usize,
+    /// Number of transactions that were unschedulable.
+    num_unschedulable: usize,
```

Comment:
Just learn more about these counters, what's the relationship between `num_received`, `num_buffered` and `num_scheduled` `num_unschedulable`? Was thinking `num_unschedulable = num_buffered - num_scheduled`, but guess that isn't correct as one tx could be unschedulable more than once?

Reply:
No explicit relation maintained between the different categories of metrics.
Transactions may be received and buffered in a different interval than they are scheduled in.
As you stated, transactions could also be unschedulable multiple times.